src/Storages/PostgreSQL/StorageMaterializedPostgreSQL.cpp

```diff
@@ -502,6 +500,9 @@ ASTPtr StorageMaterializedPostgreSQL::getCreateNestedTableQuery(
 
     create_table_query->set(create_table_query->storage, storage);
 
+    if (table_override)
+        applyTableOverrideToCreateQuery(*table_override, create_table_query.get());
```

Comment:
Hm, seems like it's possible to override `_sign` and `_version` columns now :)
But it's ok if user will get some strange error on attempt to do it.

Reply:
Added a check for it.